tidb-cloud/set-up-vpc-peering-connections.md

```diff
@@ -27,7 +27,7 @@ You can set the project CIDR when creating the first Dedicated Tier of your proj
 
     > **Note:**
     >
-    > When setting the project CIDR, avoid any conflicts with the CIDR of the VPC where your application is located.
+    > When setting the project CIDR, avoid any conflicts with the CIDR of the VPC where your application is located. You need to set a project CIDR that is different from the CIDR of the VPC where your application is located.
```

Comment:
```suggestion
    > To avoid any conflicts with the CIDR of the VPC where your application is located, you need to set a different project CIDR in this field.
```